pyproject.toml

```diff
@@ -49,6 +49,10 @@ manifest-path = "crates/ruff_cli/Cargo.toml"
 module-name = "ruff"
 python-source = "python"
 strip = true
+#exclude = [
+#  "crates/ruff/resources/test/fixtures/**/*",
+#  "crates/ruff/src/rules/*/snapshots/**/*"
+#]
```

Comment:
Nit: remove?

Reply:
oops i didn't realize i committed the comments